webapp/src/assets/svg/MLOLogo.vue

```diff
@@ -0,0 +1,225 @@
+          style="fill-rule: nonzero"
+        />
+        <path
+          :fill="color"
+          d="M1890.89,646.649C1890.89,655.097 1887.34,659.897 1879.08,659.897C1873.13,659.897 1870.15,656.825 1870.15,653.081C1870.15,647.321 1876.1,645.593 1890.89,644.057L1890.89,646.649ZM1891.94,661.049C1892.13,663.545 1892.52,666.041 1892.9,667.673L1905.1,667.673C1904.13,663.353 1903.56,658.745 1903.56,650.585C1903.56,645.881 1903.75,638.969 1903.75,637.337C1903.75,619.865 1893.86,614.681 1881.1,614.681C1867.94,614.681 1859.02,622.169 1858.06,632.345L1870.44,632.345C1870.73,626.681 1874.95,623.993 1880.62,623.993C1886.28,623.993 1890.5,626.681 1890.5,634.649L1890.5,635.609C1870.25,637.913 1856.52,641.561 1856.52,653.849C1856.52,663.161 1865.26,668.921 1875.14,668.921C1882.92,668.921 1888.3,666.521 1891.94,661.049Z"
+          style="fill-rule: nonzero"
+        />
+        <path
+          :fill="color"
+          d="M1916.62,616.025L1909.32,616.025L1909.32,625.145L1916.62,625.145L1916.62,653.753C1916.62,664.025 1918.92,667.673 1930.73,667.673L1940.42,667.673L1940.42,658.169L1936.39,658.169C1931.69,658.169 1929.86,657.113 1929.86,651.833L1929.86,625.145L1940.62,625.145L1940.62,616.025L1929.86,616.025L1929.86,602.009L1916.62,602.009L1916.62,616.025Z"
+          style="fill-rule: nonzero"
+        />
+        <path
+          :fill="color"
+          d="M1971.24,625.241C1978.92,625.241 1983.43,631.097 1983.43,641.945C1983.43,652.697 1978.92,658.553 1971.24,658.553C1963.56,658.553 1959.05,652.697 1959.05,641.945C1959.05,631.097 1963.56,625.241 1971.24,625.241ZM1971.24,668.921C1984.68,668.921 1996.87,660.089 1996.87,641.945C1996.87,623.801 1984.68,614.873 1971.24,614.873C1957.8,614.873 1945.61,623.801 1945.61,641.945C1945.61,660.089 1957.8,668.921 1971.24,668.921Z"
+          style="fill-rule: nonzero"
+        />
+        <path
+          :fill="color"
+          d="M2034.89,616.313C2034.02,616.217 2032.87,616.121 2032.01,616.121C2026.06,616.121 2021.35,618.713 2018.38,625.049L2018.38,616.025L2005.32,616.025L2005.32,667.673L2018.57,667.673L2018.57,640.601C2018.57,630.905 2022.89,628.505 2029.41,628.505C2031.34,628.505 2032.97,628.601 2034.89,628.793L2034.89,616.313Z"
+          style="fill-rule: nonzero"
+        />
+        <path
+          :fill="color"
+          d="M2047.85,684.473C2060.71,684.473 2063.4,682.169 2069.45,666.137L2088.36,616.025L2074.92,616.025L2062.92,653.273L2051.21,616.025L2036.71,616.025L2056.49,667.481L2055.91,668.921C2053.7,674.393 2051.69,674.297 2047.27,674.297L2042.28,674.297L2042.28,684.473L2047.85,684.473Z"
+          style="fill-rule: nonzero"
+        />
+      </g>
+    </g>
+  </svg>
+</template>
+
+<script lang="ts">
+export default {
+  name: "MLOLogo",
+  props: {
+    color: {
+      type: String,
+    },
+  },
+};
+</script>
```

Comment:
we prefer to use the [composition API](https://vuejs.org/guide/introduction.html#api-styles) which has [numerous advantages when used with single file components (SFC)](https://vuejs.org/api/sfc-script-setup.html). this block can be rewriten to
```suggestion
<script lang="ts" setup>
defineProps<{
  color: string;
}>();
</script>
```